tests/reactive_fluid_transport_tian_approximation_porosity.prm

```diff
@@ -0,0 +1,65 @@
+# This test checks that the tian approximation reaction model correctly 
+# outputs the melt fractions. The initial bound water is 1 wt%, and there is
+# also an initial porosity of 1% (volume fraction). 
+# The pressure is 1 GPa, and the temperature is set to 1600 K, and the solid
+# is 100% peridotite. The tian approximation says that the peridotite can only
+# store 0.10257 wt% water. The melt fraction within the tian reaction model is calculated with:
+# m_f = wt%_bound_water + wt%_free_water - equilibrium_water_content
+# So we need to compute the wt%_free_water. The porosity is a volume fraction,
+# so we just need to convert it to a mass fraction with:
+# wt%_free_water = porosity * rho_f / rho_b
+# where rho_b = (1 - porosity) * rho_s + porosity * rho_f = 2970 kg/m^3.
+# This results in an initial wt%_free_water equal to 0.0067. Therefore, we can
+# determine that the melt fraction output should be equal to:
+# m_f = 0.01 + 0.0067 - 0.0010257 = 0.01567.
```

Comment:
The actual output is 0.01585. Is this just a round-off error in your analytical computation (in which case, it would make sense to add a few more digits to compute it more accurately) or is this due to numerical error (which would be surprising, since everything is constant in the cell, so I would think the results would be very accurate).